lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -815,7 +815,7 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgSubrangeType(const DISubrange *ST) {
                          : getDebugInfoNoneId();
     }
   };
-  for (int Idx = CountIdx; Idx < OperandCount; ++Idx)
+  for (int Idx = 0; Idx < OperandCount; ++Idx)
```

Comment:
We can have less than 4 operands, as the last one is optional. Should it be handled some way here and in reverse translation?

Reply:
Moved stride to truly optional